pkg/sql/resolver.go

```diff
@@ -40,7 +40,8 @@ func (p *planner) ResolveUncachedDatabaseByName(
 	p.runWithOptions(resolveFlags{skipCache: true}, func() {
 		var desc sqlbase.DatabaseDescriptor
 		desc, err = p.LogicalSchemaAccessor().GetDatabaseDesc(
-			ctx, p.txn, p.ExecCfg().Codec, dbName, p.CommonLookupFlags(required),
+			ctx, p.txn, p.ExecCfg().Codec, dbName,
+			tree.DatabaseLookupFlags{CommonLookupFlags: p.CommonLookupFlags(required)},
```

Comment:
should make a helper that returns databaselookupflags with required

Reply:
I'll do this as part of refactoring the flags.